python/py-logbook/Portfile

```diff
@@ -19,11 +19,13 @@ long_description    {*}${description}
 
 homepage            https://packages.python.org/Logbook/
 
-checksums           rmd160  29e31018d8a3b92dfcba068de459ff7315c3ab77 \
-                    sha256  a5e8016701ca3beea6a390b0ba1541037f663543ca508ccd36cfdc841639cdd7 \
-                    size    367964
+checksums           rmd160  6bfd85b56115ecfdee31b6acd048bacab9930d26 \
+                    sha256  6d19797c1646ef2f518fe207ae88d2415487085044f5fa5f1cd725b9ab0dd329 \
+                    size    368141
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                        port:py${python.version}-cython
+                        port:py${python.version}-cython \
+                        port:py${python.version}-setuptools
```

Comment:
again, don't add the explicit `setuptools` dependency it's already done in the `python` PortGroup